lib/surrounded/access_control.rb

```diff
@@ -6,16 +6,16 @@ def self.extended(base)
         base.const_set(:AccessError, Class.new(::Surrounded::Context::AccessError))
       end
     end
-    
```

Comment:
Could these whitespace changes be due to the fact that I use a windows system and it handles line endings differently than linux? Whenever I commit, git outputs that it has changed the line endings in the commit to normalise the files for other operating systems...